components/collective-navbar/ActionsMenu.js

```diff
@@ -159,12 +260,15 @@ const CollectiveNavbarActionsMenu = ({ collective, callsToAction, hiddenActionFo
       borderTop={['1px solid #e1e1e1', 'none']}
     >
       <Box px={1}>
-        <ActionsDropdown trigger="click">
+        <HideableBox isHiddenOnMobile={hasOnlyTwoCTAs}>
+          {getSecondActionButton(secondAction.toString(), collective)}
```

Comment:
Converting the array to a string to get `hasSubmitExpense` from `["hasSubmitExpense"]` is a bit hacky. Maybe `getSecondActionButton` could directly return the right action instead of an array (or null if there's none)?